packages/plugin-ext/src/hosted/node/hosted-plugin.ts

```diff
@@ -67,7 +67,7 @@ export class HostedPluginSupport {
     }
 
     runPlugin(plugin: PluginModel): void {
-        if (plugin.entryPoint.backend) {
+        if (!plugin.entryPoint.frontend) {
```

Comment:
Can plugin have frontend and backend entry points?

Reply:
I talked with @evidolob. Theoretically could be plugin with both frontend and backend part, but we should not support this case and we can prevent setup such plugins with help some plugin validator https://github.com/theia-ide/theia/issues/4945